location/__init__.py

```diff
@@ -0,0 +1,149 @@
+import os
```

Comment:
maybe better as a util module?

Reply:
thinking we should keep the top-level for large components like search, frontend, crawl, etc.

Reply:
Agree